v2

Obywatelecki opened the pull request:
Nowy kod: Najczesciej wystepujące słowo bez html

Obywatelecki committed:
v2

jbanaszczyk commented on src/com/company/Main.java:
```diff
@@ -2,63 +2,60 @@
-                sb.append(htmlPage);
+            while ((txt = bfr.readLine()) != null) {
+                sb.append(txt);
             }
```
Sklei słowa z kolejnych linii
```
word
word
```
da `wordword`

jbanaszczyk commented on src/com/company/Main.java:
```diff
@@ -2,63 +2,60 @@
 
-            htmlPage = sb.toString().replaceAll("<[^>]*>", " ");
-            String htmlWords[] = htmlPage.split(" ");
+            String words[] = sb.toString().split(" ");
```
W tym miejscu już nie używasz pliku, to to należy go zamknąć.

jbanaszczyk commented on src/com/company/Main.java:
```diff
@@ -2,63 +2,60 @@
-            for (String word : htmlWords) {
-                if (word.length() > 3) {
+            for (String word : words) {
+                if (word.length() > 1) {
```
👍

jbanaszczyk commented on src/com/company/Main.java:
```diff
@@ -2,63 +2,60 @@
-            */
 
-            System.out.println(Collections.max(occurrences.keySet()) + " " +Collections.max(occurrences.values()));
+            int maxVal = Collections.max(occurrences.values());
```
Fajne podejście.
Nie najwydajniejsze, ale z punktu widzenia czytelności i zużycia pamięci - super :)

jbanaszczyk reviewed (approved):
Ładne :)

Obywatelecki commented:
Poprawiłem przy współpracy z A.

Dzięki!

pt., 16 mar 2018, 23:47 użytkownik Jacek Banaszczyk <
notifications@github.com> napisał:

> *@jbanaszczyk* approved this pull request.
>
> Ładne :)
> ------------------------------
>
> In src/com/company/Main.java
> <https://github.com/Obywatelecki/JavaHomework_2/pull/3#discussion_r175230624>
> :
>
> >              StringBuilder sb = new StringBuilder();
>
> -            while ((htmlPage = bfr.readLine()) != null) {
> -                sb.append(htmlPage);
> +            while ((txt = bfr.readLine()) != null) {
> +                sb.append(txt);
>              }
>
> Sklei słowa z kolejnych linii
>
> word
> word
>
> da wordword
> ------------------------------
>
> In src/com/company/Main.java
> <https://github.com/Obywatelecki/JavaHomework_2/pull/3#discussion_r175230715>
> :
>
> >              }
>
> -            htmlPage = sb.toString().replaceAll("<[^>]*>", " ");
> -            String htmlWords[] = htmlPage.split(" ");
> +            String words[] = sb.toString().split(" ");
>
> W tym miejscu już nie używasz pliku, to to należy go zamknąć.
> ------------------------------
>
> In src/com/company/Main.java
> <https://github.com/Obywatelecki/JavaHomework_2/pull/3#discussion_r175230792>
> :
>
> >
>              Map<String, Integer> occurrences = new HashMap<>();
>
> -            for (String word : htmlWords) {
> -                if (word.length() > 3) {
> +            for (String word : words) {
> +                if (word.length() > 1) {
>
> 👍
> ------------------------------
>
> In src/com/company/Main.java
> <https://github.com/Obywatelecki/JavaHomework_2/pull/3#discussion_r175231215>
> :
>
> >
> -            System.out.println(Collections.max(occurrences.keySet()) + " " +Collections.max(occurrences.values()));
> +            int maxVal = Collections.max(occurrences.values());
>
> Fajne podejście.
> Nie najwydajniejsze, ale z punktu widzenia czytelności i zużycia pamięci -
> super :)
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/Obywatelecki/JavaHomework_2/pull/3#pullrequestreview-104736920>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AYN8gg_mrgOdMKEnozP8PgiJ5H3z5zU8ks5tfED8gaJpZM4SufZX>
> .
>
